script/Arcueid/Day 7/07_00_ARC07_3_1_4.txt

```diff
@@ -38,7 +38,7 @@ But I can't turn back now that I've let slip what I'm thirsting for.
 [sha:525a02fcdcf2d44d2ee1c8481c49a5e70f35d9e5]{
 -- Page 623, Offset 18149.
 -- 「あの時は遠くにいたからよく見えなくて。
-"You were too far away back then, so I couldn't really make out what was going on.
+"You were too far away back then, so I wasn't seeing clearly.
```

Comment:
or maybe "my eyes couldn't make out what was going on", he's talking about her dress, not the fight in general

Reply:
"Wasn't seeing clearly" reads more to me as someone being blinded by passions. I'd say revert, "what was going on" is vague enough to fit with the next line.